test/bigtest/interactors/resource-edit.js

```diff
@@ -38,6 +40,7 @@ import Datepicker from './datepicker';
 }
 
 @interactor class ResourceEditDropDownMenu {
+  clickRemoveFromHoldings = clickable('.tether-element [data-test-eholdings-remove-resource-from-holdings]');
```

Comment:
Is there another attribute "data-test-eholdings-remove-resource-from-holdings"? Why it is needed to search it inside `.tether-element`?